test/httpwstest.cpp

```diff
@@ -123,10 +123,10 @@ void HTTPWSTest::testExoticLang()
 
 void HTTPWSTest::testSaveOnDisconnect()
 {
-
     const std::string testname = "saveOnDisconnect- ";
 
-    const std::string text = helpers::genRandomString(40);
+    // FIXME was helpers::genRandomString(40);
+    const std::string text = "Test string - Il<VP,|tDzyMqL:s]Ws~)hA~*v'Yy[<KwE$?`0d";
```

Comment:
core.git decides if this is markdown or not here: https://github.com/libreoffice/core/blob/eab77eea7fe93bec66bf6996fd4db2da0725073c/comphelper/source/misc/markdown.cxx#L60-L70

`patternTables->matcher(ustr, status)->find()` returns true for this input, it has enough '-' and '|' in it to match. If you drop the '|' from the text, then it won't match. I would suggest to just remove that single char here for now -- later we can consider to make that table regex more strict, too.